converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/Worker/FhirTransaction/MissingRequiredDicomTagException.cs

```diff
@@ -22,6 +23,6 @@ private static string FormatMessage(string dicomTagName)
     {
         EnsureArg.IsNotNullOrWhiteSpace(dicomTagName, nameof(dicomTagName));
 
-        return string.Format(DicomCastCoreResource.MissingRequiredDicomTag, dicomTagName);
+        return string.Format(CultureInfo.CurrentCulture, DicomCastCoreResource.MissingRequiredDicomTag, dicomTagName);
```

Comment:
I don't think we should localize these exception messages: the [Microsoft Rest API guidelines](https://github.com/Microsoft/api-guidelines/blob/vNext/Guidelines.md#7102-error-condition-responses) say "Services SHOULD NOT localize "message" for the end user, because doing so might make the value unreadable to the app developer who may be logging the value, as well as make the value less searchable on the Internet."

Reply:
That makes sense. Although our usage of a resource file is a bit misleading, as presumably we'll never add other languages. Ideally we'd always set the culture to invariant with `DOTNET_SYSTEM_GLOBALIZATION_INVARIANT` but we have to keep it disabled (`false`) for the SQL Client apparently. So instead I suppose we can just ensure we don't add any other languages to our resources and use `InvariantCulture` for formatting?

Reply:
Alternatively, we could keep the `CurrentCulture` and `.resx` for our exceptions, but explicitly set the local in our `dockerfile` like:
```dockerfile
ENV LC_ALL=en_US.UTF-8
ENV LANG=en_US.UTF-8
```

Reply:
I'd argue that we should not be applying localization anywhere - logs, error messages, responses. The Dicom standard specifies literal English strings in many cases, and all of our strings are for internal use (wouldn't be directly exposed to an end user).

Reply:
We can certainly do that for our code base, but our dependencies may include different localization. I think we even depend on certain error messages in a few cases! So I think it's best to enforce a certain locale within our application such that all dependencies are aligned with our own assemblies.